index.js

```diff
@@ -1,6 +1,7 @@
 var search = require('./search')
+var generateGrams = require('./generateGrams')
 
-module.exports = function fuzzyfind (input, collection, accessor) {
+module.exports = function fuzzyfind (input, collection, accessor, precision) {
```

Comment:
The code would look something like this:

~~~ diff
diff --git a/index.js b/index.js
index 961bde7..760ee5a 100644
--- a/index.js
+++ b/index.js
@@ -6,6 +6,10 @@ module.exports = function fuzzyfind (input, collection, accessor, precision) {
     return collection
   }
   var suggestions = []
+  if (typeof accessor === 'number') {
+    precision = accessor
+    accessor = null
+  }
   if (!accessor) {
     accessor = function(item) {
       return item
~~~